app/mailers/notify_mailer.rb

```diff
@@ -11,8 +11,15 @@ def citizen_start_email(app_id, email, application_url, client_name, provider_fi
       application_url: application_url,
       client_name: client_name,
       provider_firm: provider_firm,
-      ref_number: app_id
+      ref_number: app_id,
+      expiry_date: url_expiry_date
     )
     mail(to: email)
   end
+
+  private
+
+  def url_expiry_date
```

Comment:
Not a blocker, but I wonder if you could add this method to the `NotifyTemplateMethods` module (which is included in each of the mailers) instead of defining it in each?

Reply:
Yes, good point.